src/StatusQ/Controls/StatusBaseInput.qml

```diff
@@ -56,6 +56,23 @@ Item {
         color: Theme.palette.baseColor1
     }
 
+    property Item component
+
+    onClearableChanged: {
+        if (clearable && !component) {
+            clearButtonLoader.active = true
+            clearButtonLoader.parent = statusBaseInputComponentSlot
+        } else {
+            clearButtonLoader.active = false
+        }
```

Comment:
Soo, if user set up component and clearable - component have a high priority?

Reply:
If yes its not clear.

Reply:
Yes, if a `component` is given, `clearable` is ignored (because then there's a conflict, two components try to get rendered in the same spot)